letsencrypt/auth_handler.py

```diff
@@ -139,11 +154,13 @@ def _respond(self, cont_resp, dv_resp, best_effort):
 
         # Check for updated status...
         try:
-            self._poll_challenges(chall_update, best_effort)
+            result = self._poll_challenges(chall_update, best_effort)
```

Comment:
should we call this failed_domain_list or failed_domains or something? It's a bit oblique at this point what result is expected to be